examples/index.md

```diff
@@ -40,7 +40,7 @@ client.fetchProduct('your-product-id').then(function(product) {
   "<img class='product__image' src='" + product.selectedVariantImage.src + "' >" +
   "<h2 class='product__title'>" + product.title + "</h2>" +
   "<a class='product__buy' href='" +
-    product.selectedVariant.checkoutUrl({quantity: 1}) +
+    product.selectedVariant.checkoutUrl([quantity = 1]) +
```

Comment:
Since this is an example, this should probably just be 
```javascript
product.selectedVariant.checkoutUrl()
```
Or, if we want to be explicit about the quantity:
```javascript
product.selectedVariant.checkoutUrl(1)
```